### PR TITLE
CI fixes (wip)

### PR DESCRIPTION
**Pull request by shin-**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -28,7 +28,6 @@ integration-test-py3: build-py3
 	docker run -v `$(HOST_TMPDIR)`:/tmp -v /var/run/docker.sock:/var/run/docker.sock docker-py3 py.test -rxs tests/integration_test.py
 
 integration-dind: build build-py3
-	docker build -t dpy-tests -f ./tests/Dockerfile .
```
Comment (dnephin) [inline Makefile]: Is this being done as part of the jenkins config? Maybe just change this to a comment so that users can run it locally ?

Comment (shin-) [inline Makefile]: No, it's just unneeded, and I forgot to remove it - we already have Dockerfiles for building docker-py at the root.

Comment (dnephin) [inline Makefile]: Ah I see, so should the `integration-dind` target depend on `build` and `build-py3` ?

Comment (shin-) [inline Makefile]: yep! Already does!

Comment (dnephin) [inline Makefile]: oops, right!

**Comment by dnephin** [comment]
LGTM

**Pull request merged**